docs/install.md

```diff
@@ -107,13 +107,20 @@ Please install the following:
 On Debian or Ubuntu, you should be able to get all these with:
 
     sudo apt-get install build-essential libcap-dev xz-utils zip \
-        unzip imagemagick strace curl clang-3.4 discount
+        unzip imagemagick strace curl clang-3.4 discount git
     curl https://install.meteor.com/ | sh
 
+### Get the source code:
+
+Get the source code from the git repository
```

Comment:
Oh I see! You added it to the section heading but I meant to the sentence below the section heading. So one more change requested:

remove the `:` from section heading, and add it to the sentence below the heading.